packages/connect/src/utils/firmwareReleaseConfigUtils.ts

```diff
@@ -6,122 +6,119 @@ import { getJWSPublicKey } from '@trezor/env-utils';
 import { firmwareReleaseConfigAssets } from './assetUtils';
 import { FirmwareUpdateSource, getOnlineFirmwareBaseUrl } from '../data/firmwareInfo';
 
-const JWS_SIGN_ALGORITHM = 'ES256';
-const VERSION = 1;
-const JWS_RELEASES_FILENAME_REMOTE = `releases.v${VERSION}.json`;
+const JWS_CONFIG = {
+    SIGN_ALGORITHM: 'ES256',
+    VERSION: 1,
+    REMOTE_FILENAME: 'releases.v1.json',
+    REQUEST_TIMEOUT_MS: 5000,
```

Comment:
I'd use camel case for these:
```suggestion
    signAlgorithm: 'ES256',
    version: 1,
    remoteFilename: 'releases.v1.json',
    requestTimeoutMs: 5000,
```